crates/ursa-utils/src/contract_bindings/epoch.rs

```diff
@@ -0,0 +1,652 @@
+pub use epoch::*;
```

Comment:
Whats this file

Reply:
This is autogenerated file from ethers that handles contract bindings. Very useful for taking bytes returned from evm and turning into rust structs/ vice versa. I can extract just the stuff we are using currently to clean it up a bit if you want

Reply:
Removed and replaced the auto generated files